app/src/main/java/com/duckduckgo/app/referral/AppReferrerDataStore.kt

```diff
@@ -29,6 +29,7 @@ interface AppReferrerDataStore {
     var campaignSuffix: String?
     var installedFromEuAuction: Boolean
     var utmOriginAttributeCampaign: String?
+    var returningUser: Boolean
```

Comment:
Why are we duplicating `returningUser` storage? now we have two sources of truth